ui/i18n.js

```diff
@@ -0,0 +1,31 @@
+
+// cross-browser connection to extension i18n API
+
+const chrome = chrome || null
+const browser = browser || null
+let getMessage = null
+
+if ((chrome && chrome.i18n && chrome.i18n.getMessage) ||
+    (browser && browser.i18n && browser.i18n.getMessage)) {
```

Comment:
I would slightly prefer using [extensionizer](https://www.npmjs.com/package/extensionizer) to this inline logic.